guests/templates/checklist/merch_deadline.html

```diff
@@ -23,77 +21,36 @@
-      merchandise throughout the festival.{% endif %}
-    </p>
-
-    {% if c.ROCK_ISLAND %}
-      <p>
-        It is recommended, but not required, for performers to use our Rock Island service in this area, as merchandise can
-        be sold more consistently, and for many more hours.
-      </p>
-      <p>
-        <a href="../static/RockIsland.pdf">
-          Click here for more info about our Rock Island service!
-        </a>
-      </p>
-    {% endif %}
-
-    {% if c.REQUIRE_DEDICATED_GUEST_TABLE_PRESENCE %}
-      <p>
-        If you elect to{% if c.ROCK_ISLAND %} decline the use of Rock Island and{% endif %} sell your merchandise directly,
-        we require that your table be staffed for at least <b>8 hours per day</b>.
-      </p>
-    {% endif %}
-
-    <p>
-      On the night of the performance, table space will also be provided immediately outside the concert area for
-      merchandise sales and signings.
-    </p>
+    Space will be provided in our marketplace area for merchandise sales.
   {% endblock %}
 
   <br/> <br/>
 
-  <form method="post" action="merch">
+  <form method="post" action="merch" class="form-horizontal">
     {{ csrf_token() }}
     <input type="hidden" name="guest_id" value="{{ guest.id }}" />
     <input type="hidden" name="id" value="{{ guest_merch.db_id }}" />
-    <select name="selling_merch">
-      <option value="">Tell us how you'd like to sell your merch</option>
-      {{ options(c.GUEST_MERCH_OPTS,guest_merch.selling_merch) }}
-    </select>
-    <br/> <br/>
-    {% if c.REQUIRE_DEDICATED_GUEST_TABLE_PRESENCE %}
-      <div id="coverage" style="display:none">
-        <input type="checkbox" name="coverage" value="yes" />
-        I commit to staffing our table in the marketplace at least <b>8 hours per day</b>.
-        <br/> <br/>
-        <input type="checkbox" name="warning" value="yes" />
-        I acknowledge that failing to staff our table for <b>8 hours per day</b> will revoke the privilege of selling our merchandise from our own dedicated table in the future
-        <br/> <br/>
+
+    <div class="form-group">
+      <label class="col-sm-3 control-label">Merch Options</label>
+      <div class="col-sm-6">
+        <select name="selling_merch" id="selling_merch" class="form-control">
+          <option value="">Tell us how you'd like to sell your merch</option>
+          {{ options(c.GUEST_MERCH_OPTS,guest_merch.selling_merch) }}
+        </select>
       </div>
-    {% endif %}
-    {% block form_extra %}{% endblock %}
+    </div>
+    {% block form_extra %}
+      {% if c.ROCK_ISLAND %}
+        {# c.ROCK_ISLAND is only for bands, but it's easier to remove it from the template than to mess with the
+         actual GUEST_MERCH options. If using form_extra here for other groups, call {{ super() }} #}
+        <script type="text/javascript">
+            $(function () {
+                $("#selling_merch option[value='{{ c.ROCK_ISLAND }}']").remove();
```

Comment:
I am not a huge fan of this style of post-hoc manipulation of HTML that could be rendered by Jinja.

One alternative solution might look like this:
```
<select name="selling_merch" id="selling_merch" class="form-control">
  <option value="">Tell us how you'd like to sell your merch</option>
  {% if guest.group_type == c.BAND %}
    {{ options(c.GUEST_MERCH_OPTS, guest_merch.selling_merch) }}
  {% else %}
    {{ options(c.GUEST_MERCH_OPTS|rejectattr('0', "equalto", c.ROCK_ISLAND), guest_merch.selling_merch) }}
  {% endif %}
</select>
```

However, I it may be even better to specify this in the config? I am unsure of this, and don't have strong feelings either way. But it might look something like this:
```
[enums]
[[band_merch]]
no_merch = "Not selling merch"
own_table = "Dedicated table"
rock_island = "Rock Island"

[[guest_merch]]
no_merch = "Not selling merch"
own_table = "Dedicated table"
```

Reply:
I wanted to avoid two separate enums, since they use the same DB column and that would get messy. But I like the Jinja filter method!